### ISSUE TITLE
Cannot convert a symbolic Tensor (lstm/strided_slice:0) to a numpy array

### ISSUE THREAD

**Issue by msh-nayan** [chaquo/chaquopy#570]
#### Chaquopy version

10.0.1 (paid version)

#### Devices or emulators where the issue happens

Physical devices

#### Relevant parts of your code

(1) root label build.gradle file

```
dependencies {
        classpath "com.android.tools.build:gradle:4.1.3"
        classpath "com.chaquo.python:gradle:10.0.1"
}
```
(2) app label build.gradle file 
```
plugins {
    id 'com.android.application'
    id 'com.chaquo.python'
}
```

```
sourceSets {
            main {
                python {
                    srcDirs = ["src/main/python"]
                }
            }
        }
        python {
            pip {
                install "numpy"
//                install "numpy==1.17.4"
                install "pandas"
                install "scikit-learn"
                install "tensorflow"
                install "h5py"
            }
        }
        ndk {
            abiFilters "armeabi-v7a", "arm64-v8a", "x86", "x86_64"
        }
```

(3) Python build version 3.8

```
#####:~$ python3 -V
Python 3.8.0
```

#### Describe your issue

when I use "numpy==1.17.4", everything is working as expected. I've tested on several devices, it is working. 

But when I use "numpy" (without mentioning it's version), the following error happens on every devices that I tested.

**Cannot convert a symbolic Tensor (lstm/strided_slice:0) to a numpy array**

Your suggestion will be helpful to fix this issue. 
Thanks for your support though.

**Comment by mhsmith** [comment]
Please post the full stack trace, and the relevant parts of your Python code.

**Comment by mhsmith** [comment]
I think other users may have the same problem, so it would be very helpful if you let me know the stack trace and the relevant lines of your Python code. You can [email them](https://chaquo.com/chaquopy/contact/) to me if you prefer.

**Comment by msh-nayan** [comment]
Thanks for the support. 
As right now, it is working with numpy 1.17.4 version, so once I'm relatively free I'll run the code using "numpy" (without mentioning version) and after that I'll share stack trace & relevant Python code. This may take a while :)

**Comment by mhsmith** [comment]
Thanks very much.

**Comment by mhsmith** [comment]
I've managed to reproduce it with the following code (from [here](https://stackoverflow.com/questions/68378097)):

```
from tensorflow import keras
model = keras.Sequential()
model.add(keras.layers.LSTM(85, input_shape=(1, 53)))
```

**Comment by mhsmith** [comment]
NumPy 1.21.4 is incompatible with TensorFlow 2.1.0 (https://github.com/tensorflow/tensorflow/issues/47691), so we've removed it from the default repository and replaced it with NumPy 1.19.5.

If you need NumPy 1.21.4 for any reason, you can enable it by adding this line to the `pip` block in your build.gradle file:

    options "--extra-index-url", "https://chaquo.com/pypi-extra/"
